website/docs/en/guide/optimization/analysis.mdx

```diff
@@ -11,3 +11,32 @@ You can also generate a `stats.json` file for further analysis with other bundle
 ```sh
 $ rspack build --json stats.json
 ```
+
+## Use Rsdoctor
+
+**Rsdoctor** provides the `Bundle Size` module, which is mainly used to analyze the information of the build artifacts of **Webpack** or **Rspack**, including the **size of resources**, **duplicate packages**, and **module reference relationships**:
+
+- **Bundle Overview**: Displays the total number and size of artifacts, as well as the number and size of each file type. It also shows the duplicate packages and their reference chains.
+- **Bundle Analysis Module**: Analyzes the size and code information of the build artifacts' resources (**Assets**) and the included **Modules**. In this module, you can view the **actual code size of modules after packaging** in the Assets, as well as the original code or **packaged code segments** and **module reference relationships**.
+
+<img
+  src="https://lf3-static.bytednsdoc.com/obj/eden-cn/lognuvj/rsdoctor/docs/usage/bundle/bundle-size-overall.png"
```

Comment:
Please upload all images to https://github.com/rspack-contrib/rstack-design-resources